contracts/StandardTokenGateway.sol

```diff
@@ -96,10 +103,22 @@ contract StandardTokenGateway is AccessControl {
 		_;
 	}
 
-	function getStandardTokenPrice() public view returns (uint256, bool) {
+	function getSeuroStandardTokenPrice() public view returns (uint256, bool) {
 		return (tokenPrice, inversed);
 	}
 
+    function seuroToStandardToken(uint256 _seuroAmount) public view returns (uint256) {
+        return inversed ?
+            _seuroAmount * tokenPrice :
+            _seuroAmount / tokenPrice;
+    }
+
+    function otherToStandardToken(uint256 _otherAmount) external view returns (uint256) {
+        (,int256 eurOtherRate,,,) = IChainlink(chainlinkEurOther).latestRoundData();
+        uint256 seuroValue = _otherAmount * 10 ** chainlinkDec / uint256(eurOtherRate);
```

Comment:
I was thinking we make the gateway a hub for rewards. So we let many different reward bonds (i.e., type and decimals) interact with the gateway. Hence, the CL decimals must be flexible and we ideally don't re-deploy it unless critical.

Can you parametrise the decimals instead?

Reply:
ok yeah i think i understand what you mean